guides/source/active_record_querying.md

```diff
@@ -24,32 +24,101 @@ Code examples throughout this guide will refer to one or more of the following m
 
 TIP: All of the following models use `id` as the primary key, unless specified otherwise.
 
+
 ```ruby
-class Client < ApplicationRecord
-  has_one :address
-  has_many :orders
-  has_and_belongs_to_many :roles
+class Author < ApplicationRecord
+  has_many :books, -> { order('year_published DESC') }
```

Comment:
Is the default order necessary? If can affect other example queries.

Reply:
I'd like to leave it in because it ensures the order of the results are deterministic; that's really helpful to a novice looking at a list of result -- the novice doesn't have to wonder if something is wrong if their results aren't in the same order.
Also it really helps with the _readability_ of the results in queries:  the results appear in an order that makes sense at a glance, so it is (cognitively) easier to look at and understand the data.

Reply:
What about to mention and explain why random ordering is possible in examples instead of making default order in `has_many` relation? That will be maybe more "real world" example. I'm not sure how common this pattern is, but I have never used that.